sdk/attestation/Azure.Security.Attestation/src/AttestationToken.cs

```diff
@@ -448,7 +448,11 @@ private bool ValidateTokenSignature(AttestationSigner[] possibleSigners)
             {
                 // The leaf certificate is defined as the certificate which signed the token, so we just need to look
                 // at the first certificate in the chain.
+#if NET6_0_OR_GREATER
```

Comment:
Probably want to bounce this off of Larry, just to be sure we didn't make an unintentional behavior change.

Reply:
cc:/ @LarryOsterman